ci/jenkins/jobs/projects-lab.yaml

```diff
@@ -1313,3 +1313,113 @@
               default-excludes: true
               fingerprint: false
               only-if-success: false
+      - '{name}-{test_name}-test-ipv6-ds':
```

Comment:
We should name it '{name}-{test_name}-for-pull-request' if you want to trigger this job in PR.